app/controllers/admin_controller.rb

```diff
@@ -51,8 +51,10 @@ def projects
       c << ['LOWER(identifier) LIKE ? OR LOWER(name) LIKE ?', name, name]
     end
 
-    @projects = Project.find :all, order: 'lft',
-                                   conditions: c.conditions
+    @projects = Project.with_required_storage
+                .find :all,
+                      order: 'lft',
+                      conditions: c.conditions
```

Comment:
I left this in there because https://github.com/opf/openproject/pull/3344 touches it again immediately afterwards.